package.json

```diff
@@ -38,14 +37,11 @@
     "ycsb": "node ./benchmark/ycsb.js  run -P ./benchmark/workloada -p table=usertable -p cloudspanner.instance=ycsb-instance -p operationcount=100 -p cloudspanner.database=ycsb",
     "fix": "gts fix && eslint --fix '**/*.js'",
```

Comment:
we shouldn't need the `eslint` any longer with the new `gts`.

Reply:
Removing it!